src/include/self_driving/modeling/operating_unit_defs.h

```diff
@@ -108,6 +108,10 @@ enum class ExecutionOperatingUnitType : uint32_t {
    * translator (which requires an OpUnit type). We skip this OpUnit in the OperatingUnitRecorder.
    */
   LIMIT,
+  /**
+   * ANALYZE
```

Comment:
Can you make an issue for this an point it to Lin? I am not sure if they want to record metrics for Analyze queries, but if not, it would be nice to have a comment mentioning that Analyze OpUnits are being skipped in the recorder as above.

Reply:
https://github.com/cmu-db/noisepage/issues/1488

Reply:
Please see my response in #1488. Basically adding a comment here would be fine.

Reply:
Also added a comment